Vungle/VungleRouter.h

```diff
@@ -38,19 +39,21 @@ extern const CGSize kVNGLeaderboardBannerSize;
 - (void)requestInterstitialAdWithCustomEventInfo:(NSDictionary *)info delegate:(id<VungleRouterDelegate>)delegate;
 - (void)requestRewardedVideoAdWithCustomEventInfo:(NSDictionary *)info delegate:(id<VungleRouterDelegate>)delegate;
 - (void)requestBannerAdWithCustomEventInfo:(NSDictionary *)info size:(CGSize)size delegate:(id<VungleRouterDelegate>)delegate;
-- (BOOL)isAdAvailableForPlacementId:(NSString *)placementId;
+- (BOOL)isAdAvailableForDelegate:(id<VungleRouterDelegate>)delegate;
 - (NSString *)currentSuperToken;
-- (void)presentInterstitialAdFromViewController:(UIViewController *)viewController options:(NSDictionary *)options forPlacementId:(NSString *)placementId;
-- (void)presentRewardedVideoAdFromViewController:(UIViewController *)viewController customerId:(NSString *)customerId settings:(VungleInstanceMediationSettings *)settings forPlacementId:(NSString *)placementId;
+- (void)presentInterstitialAdFromViewController:(UIViewController *)viewController options:(NSDictionary *)options delegate:(id<VungleRouterDelegate>)delegate;
+- (void)presentRewardedVideoAdFromViewController:(UIViewController *)viewController customerId:(NSString *)customerId settings:(VungleInstanceMediationSettings *)settings delegate:(id<VungleRouterDelegate>)delegate;
 - (UIView *)renderBannerAdInView:(UIView *)bannerView
                         delegate:(id<VungleRouterDelegate>)delegate
                          options:(NSDictionary *)options
                   forPlacementID:(NSString *)placementID
                             size:(CGSize)size;
+- (void)completeBannerAdViewForDelegate:(id<VungleRouterDelegate>)delegate;
 - (void)updateConsentStatus:(VungleConsentStatus)status;
 - (VungleConsentStatus) getCurrentConsentStatus;
-- (void)clearDelegateForPlacementId:(NSString *)placementId;
+- (void)cleanupFullScreenDelegate:(id<VungleRouterDelegate>)delegate;
```

Comment:
Renamed because `clearDelegateForPlacementId` doesn't clear the delegate if the placement id is for a banner. What the functionality does is to remove the delegate from the interstitial / rewarded delegate tracking map.